src/main/java/redis/clients/jedis/BinaryJedisCommands.java

```diff
@@ -23,7 +23,7 @@
 
   Long expire(byte[] key, int seconds);
 
-  Long pexpire(final String key, final long milliseconds);
+  Long pexpire(byte[] key, final long milliseconds);
```

Comment:
@taer 
There're two pexpire() exists which are same.
You can just remove this line to continue.